src/qt/locale/bitcoin_nl.ts

```diff
@@ -1457,45 +1434,33 @@
         <source>Always do your own research before using an external cryptocurrency service</source>
         <translation>Doe altijd je eigen onderzoek voordat je een externe cryptocurrency-service gebruikt</translation>
     </message>
-    <message>
-        <source>Never share your private key to an untrustworthy person.</source>
-        <translation type="vanished">Deel uw privésleutel nooit met een onbetrouwbaar persoon.</translation>
-    </message>
-    <message>
-        <source>Who own the private keys own the coins.</source>
-        <translation type="vanished">Wie de privésleutels bezit, bezit de munten.</translation>
-    </message>
-    <message>
-        <source>To see ongoing development and contribute, checkout Dogecoin repository on GitHub!</source>
-        <translation type="vanished">Bekijk de Dogecoin-repository op GitHub om de lopende ontwikkeling te zien en bij te dragen!</translation>
-    </message>
     <message>
         <source>Services that claim to double your dogecoins are always ponzi schemes</source>
         <translation>Diensten die beweren uw dogecoins te verdubbelen zijn altijd ponzifraude</translation>
     </message>
     <message>
         <source>Never share your wallet.dat file with anyone</source>
-        <translation type="unfinished"></translation>
+        <translation>Deel uw wallet.dat bestand nooit met iemand</translation>
     </message>
     <message>
         <source>For advanced operations, use the console in &apos;Help&apos; -&gt; &apos;Debug Window&apos;</source>
-        <translation type="unfinished"></translation>
+        <translation>Gebruik voor meer geavanceerde functies de console in &apos;Help&apos; -&gt; &apos;Foutopsporingsvenster&apos;</translation>
     </message>
     <message>
         <source>Make sure to keep your wallet software updated</source>
-        <translation type="unfinished"></translation>
+        <translation>Zorg ervoor dat u uw portemonnee software up-to-date houdt</translation>
     </message>
     <message>
         <source>Never share your private key with anyone</source>
-        <translation type="unfinished"></translation>
+        <translation>Deel uw privésleutel nooit met iemand</translation>
     </message>
     <message>
         <source>Who owns the private keys, owns the coins</source>
-        <translation type="unfinished"></translation>
+        <translation>Wie de privésleutels bezit, bezit de munten</translation>
     </message>
     <message>
         <source>To see ongoing development and contribute, check out the Dogecoin Core repository on GitHub</source>
-        <translation type="unfinished"></translation>
+        <translation>Bekijk de Dogecoin Core repository op GitHub om de lopende ontwikkeling te zien en bij te dragen!</translation>
```

Comment:
Om lopende ontwikkeling en bijdrage te zien, bekijk de Dogecoin Core repository op GitHub

Reply:
This would change the meaning. I can change to passive form though.

Reply:
Om lopende ontwikkeling te zien en een bijdrage te geven, bekijk de Dogecoin Core repository op GitHub